instagram.user.styl

```
@@ -105,13 +110,26 @@ rad() { border-radius: arguments }
         --cyan      : _cyan;
         --green     : _green;
 
+        // Enable theme scrollbar width
+        if ( custom_width == 1) {
+            /::-webkit-scrollbar {
+                width: .5em
+            }
```

This enables thin scrollbars in Firefox.

```suggestion
            /::-webkit-scrollbar { width: .5em }
            /\* { scrollbar-width: thin }
```